sw/device/tests/entropy_src_fw_override_test.c

```diff
@@ -55,6 +56,11 @@ enum {
    * and blocks the ISR.
    */
   kMaxOutputWords = 16,
+  /**
+   * Number of micro seconds we wait between having extracted words from the
+   * observe FIFO and inserting words back into the entropy source.
+   */
+  kOutputdelay = 128,
```

Comment:
Don't we usually add the unit for such constants? I.e., should this be `kOutputDelayUs`?

Reply:
You're right, I'll add the unit.